consul_exporter.go

```diff
@@ -173,38 +187,66 @@ func (e *Exporter) queryClient(services chan<- []*consul_api.ServiceEntry) {
 
 		services <- s_entries
 	}
+
+	c_entries, _, err := e.client.Health().State("any", &consul_api.QueryOptions{})
+	if err != nil {
+		log.Errorf("Failed to query service health: %v", err)
+
+	} else {
+		checks <- c_entries
+	}
+
 }
 
-func (e *Exporter) setMetrics(services <-chan []*consul_api.ServiceEntry) {
+func (e *Exporter) setMetrics(services <-chan []*consul_api.ServiceEntry, checks <-chan []*consul_api.HealthCheck) {
```

Comment:
I think we should actually just synchronously set metrics from `queryClient` instead of doing this channel communication between two goroutines. AFAICS it doesn't add meaningful parallelism anyways, since one is simply synchronously waiting for the other all the time.

But that change doesn't have to go into this PR anymore...